@navikt/core/react/src/util/hooks/descendants/useDescendant.tsx

```diff
@@ -38,7 +38,7 @@ export function createDescendantContext<
    * - ref callback to register the descendant
    * - Its enabled index compared to other enabled descendants
    */
-  function _useDescendant(options?: DescendantOptions<K>) {
+  function UseDescendant(options?: DescendantOptions<K>) {
```

Comment:
Hooks should start with lowercase letter (same for UseDescendants)
```suggestion
  function useDescendant(options?: DescendantOptions<K>) {
```